css/clubs.scss

```diff
@@ -49,6 +49,10 @@
     * {
       color: black;
     }
+
+    a {
+      color: #9d68dc;
```

Comment:
Was it an active decision to not use/define $TeX-Purple everywhere?

Reply:
I haven't found a way to globally define $TeX-Purple in SCSS. Happy to make these changes if you know how

Reply:
Fine for now, but can raise an issue to refactor colour variables as tech debt